templates/etc_hosts.j2

```diff
@@ -19,8 +19,25 @@ ff02::2  ip6-allrouters
 
 # Entries for Ansible managed hosts
 {% for group in hosts_add_ansible_managed_hosts_groups %}
-{% for host in groups[group] if hostvars[host][hosts_ansible_network_interface] is defined %}
-{{ hostvars[host][hosts_ansible_network_interface][hosts_ip_protocol]['address'] }} {{ hostvars[host]['ansible_fqdn'] }} {{ hostvars[host]['ansible_hostname'] }}
+{% for host in groups[group]|sort %}
```

Comment:
Even if you don't want to take this PR, you should at least add the `sort` here. Otherwise, the order is random and the hosts file is usually listed by ansible as `changed` 👎